src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -111,4 +128,42 @@ public void println(String string) {
             super.println(string);
         }
     }
+
+    public static class MultiOutputStream extends OutputStream {
```

Comment:
While the impl is generally fine, there's one critical issue, if any output stream fails all others will not get the data at all.

So if the 1st output throws an IOException, no other one gets the data.

You should try-catch all calls in the loop, and then maybe either bubble-up the exception, or wrap multiple exceptions around another one to bubble them up, unsure

Reply:
This continues being a relevant point